sdk/include/psdk/ntsecapi.h

```diff
@@ -228,7 +229,8 @@ typedef enum _MSV1_0_PROTOCOL_MESSAGE_TYPE {
   MsV1_0DeriveCredential,
   MsV1_0CacheLookup,
   MsV1_0SetProcessOption
-} MSV1_0_PROTOCOL_MESSAGE_TYPE, *PMSV1_0_PROTOCOL_MESSAGE_TYPE;
+} MSV1_0_PROTOCOL_MESSAGE_TYPE,
+    *PMSV1_0_PROTOCOL_MESSAGE_TYPE;
```

Comment:
broken.

Reply:
Travis CI wants it that way ... should i revert and ignore travis?

Reply:
Yes at this moment we can ignore it.

Reply:
I've reverted the code style fixes. (added as new commit).